ResearchKit/Common/ORKChoiceAnswerFormatHelper.m

```diff
@@ -170,4 +174,15 @@ - (NSArray *)selectedIndexesForAnswer:(nullable id)answer {
     
 }
 
+- (NSString *)stringForChoiceAnswer:(id)answer {
+    __block NSMutableArray<NSString *> *answerStrings = [[NSMutableArray alloc] init];
```

Comment:
Do we need __block?
The array only lives within the scope of this function. The return type is a string using the array. And there is no block code within the function.